src/main/java/org/kohsuke/github/GHOrganization.java

```diff
@@ -367,6 +370,35 @@ public void conceal(GHUser u) throws IOException {
         root.createRequest().method("DELETE").withUrlPath("/orgs/" + login + "/public_members/" + u.getLogin()).send();
     }
 
+    /**
+     * Are projects enabled for organization boolean.
+     *
+     * @return the boolean
+     */
+    public boolean areProjectsEnabledForOrganization() {
+        return has_organization_projects;
+    }
```

Comment:
I like having more meaningful names, but it looks like we didn't do that for repository:

https://github.com/hub4j/github-api/blob/b23934a5a15ccc4964a20a7ab422ee894dd9bbfb/src/main/java/org/kohsuke/github/GHRepository.java#L619-L621

I'd suggest the following to keep the same pattern. 

```suggestion
    public boolean hasOrganizationProjects() {
        return has_organization_projects;
    }
```

However, I'd also be okay with the following: 

```suggestion
    public boolean areOrganizationProjectsEnabled() {
        return has_organization_projects;
    }
```

I'm also open to discussion.

Reply:
I understand consistency in naming to reduce overhead in remembering API names for the clients. However, the name `hasOrganizationProjects` answers the question whether the `Organization` has projects rather than _are projects enabled for an organization_ . 
Your second suggestion is definitely an improvement over the name I've given and will run with it.